backend/src/apiserver/storage/pipeline_store.go

```diff
@@ -48,7 +51,10 @@ func (s *PipelineStore) ListPipelines(opts *list.Options) ([]*model.Pipeline, in
 	}
 
 	buildQuery := func(sqlBuilder sq.SelectBuilder) sq.SelectBuilder {
-		return sqlBuilder.From("pipelines").Where(sq.Eq{"Status": model.PipelineReady})
+		return sqlBuilder.
+			From("pipelines").
+			LeftJoin("pipeline_versions ON pipelines.DefaultVersionId = pipeline_versions.UUID").
+			Where(sq.Eq{"pipelines.Status": model.PipelineReady})
 	}
 
 	sqlBuilder := buildQuery(sq.Select("*"))
```

Comment:
the select (*) won't guarantee the order of the columns returned. in case you change the order of the pipeline model field, it might cause the row scanning fail. 

With the upcoming breaking DB schema change, this might be an issue. 

Consider following this sample, and explicitly select the order of columns that is same as the order in the rows.scan() 
https://github.com/kubeflow/pipelines/blob/master/backend/src/apiserver/storage/run_store.go#L171

Reply:
We had a few other places in the codebase uses select * and those would need to be changed too. we can do separately.

Reply:
OK